static/src/styles/landing/LandingAbout.styl

```diff
@@ -1,3 +1,42 @@
 .LandingAbout {
   padding: grid(4);
 }
+
+.LandingAbout__imageContainer {
+  flex: 1;
+}
+
+@media screen and (min-width: 700px) {
+  .LandingAbout {
+    display: flex;
+  }
+  
+  .LandingAbout__imageContainer {
+    padding: grid(5);
+  }
+}
+
+.LandingAbout__image {
+  width: 100%;
+}
+
+.LandingAbout__text {
+  flex: 1;
+}
+
+.LandingAbout__subtitle {
+  // TODO: Remove this when Text--sectinoHeading is merged in
+  font-family: $Text__primaryFont;
+  text-transform: uppercase;
+  letter-spacing: 0.15em;
+  font-weight: 500;
+  Text__headerScale(0);
+  color: $Color__mediumGrayText;
+}
+
+.LandingAbout__headline {
+  Text__headerScale(2);
+  font-family: $Text__monospacedFont;
+  text-transform: uppercase;
```

Comment:
Do we want this to be uppercase? The monospace font looks especially blocky in all caps.

Reply:
I'm not sure about this either, I was just going by the design for now. Lowercase also didn't look great though. Will play around